cutest_kitten.js

```diff
@@ -0,0 +1,63 @@
+$(function() {
+
+  var Photo = function () {
+    this.votes = 0;
+  }
+
+  var kittens = [];
+
+  for (var i = 0; i < 14; i++) {
+    kittens.push(new Photo(i));
```

Comment:
Ah, good catch.  That was left over from when I was still using "i" to generate my kitten names.  
 
Date: Fri, 29 May 2015 15:53:50 -0700
From: notifications@github.com
To: vote_tracker@noreply.github.com
CC: hollislau@hotmail.com
Subject: Re: [vote_tracker] Add JS and jQuery code for voting app functionality; add HTML and CSS for basic styling (#1)

In cutest_kitten.js:

> @@ -0,0 +1,63 @@
> +$(function() {
> +
> +  var Photo = function () {
> +    this.votes = 0;
> +  }
> +
> +  var kittens = [];
> +
> +  for (var i = 0; i < 14; i++) {
> +    kittens.push(new Photo(i));

Is the i parameter you are passing in doing anything?


—
Reply to this email directly or view it on GitHub.